collections/misc/collectionproperties.php

```diff
@@ -128,8 +130,8 @@ function verifyAddContactForm(f){
 				</div>
 			</fieldset>
 
-			sesarTools', 'IGSN Profile
-			labelFormat
+			<?php echo $LANG['SESATOOLS']; ?>', '<?php echo $LANG['IGSN_PROFILE']; ?>
```

Comment:
I think these are for future reference, not actual lang tags. You can remove them and the LABEL FORMAT one